validation/schema.json

```
@@ -286,6 +286,21 @@
             "items": {
               "type": "string"
             }
+          },
+          "type": {
+            "type": "string",
+            "enum": [
+              "FINDER",
+              "REPORTER",
+              "ANALYST",
+              "COORDINATOR",
+              "REMEDIATION DEVELOPER",
+              "REMEDIATION REVIEWER",
+              "REMEDIATION VERIFIER",
```

Since this is intended to be machine readable, should we encode these keywords instead of a phrase?

```suggestion
              "REMEDIATION_DEVELOPER",
              "REMEDIATION_REVIEWER",
              "REMEDIATION_VERIFIER",
```